spark-doris-connector/src/main/scala/org/apache/doris/spark/sql/DorisSourceProvider.scala

```diff
@@ -83,10 +84,11 @@ private[sql] class DorisSourceProvider extends DataSourceRegister
           line.add(field.asInstanceOf[AnyRef])
         }
         rowsBuffer.add(line)
-        if (rowsBuffer.size > maxRowCount) {
+        if (!partitionTaskAtomicity && rowsBuffer.size > maxRowCount) {
           flush
         }
       })
+      flush
```

Comment:
What does this flush deal with？

Reply:
This flush is to allow each partition to commit only one StreamLoad task when the partitionTaskAtomicity parameter is true, but seems redundant because the code already has the "if (! rowArray.isEmpty) {flush} "operation.